### PR TITLE
preview merge from main

### PR DESCRIPTION
**Pull request by zhiyuanliang-ms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,10 +139,10 @@ A `RequirementType` of `All` changes the traversal. First, if there are no filte
 ```
 
 In the above example, `FeatureW` specifies a `RequirementType` of `All`, meaning all of it's filters must evaluate to true for the feature to be enabled. In this case, the feature will be enabled for 50% of users during the specified time window.
+<<<<<<< HEAD
```
Comment (jimmyca15) [inline README.md:142]: .
```suggestion


```

Comment (zhiyuanliang-ms) [reply README.md]: Oh, sorry for missing that.

**Review by jimmyca15 — APPROVED** [review]

**Comment by jimmyca15** [comment]
@zhiyuanliang please complete with a "merge" commit rather than a squash commit

**Comment by jimmyca15** [comment]
@zhiyuanliang-ms please complete with a "merge" commit rather than a squash commit

**Pull request merged**